app/views/users/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<%if @user.errors.any?%>
+  <p>The fields are mandotory</p>
+  <% @user.errors.full_messages.each do |e|%>
+    <ul style="color:red"><%= e %></ul>
+    <% end %>
+<% end%>
+
+
+<%= form_for @user do |f| %>
+  Username:</br><%= f.text_field :username, placeholder: "Your username" %></br>
+  Email:</br> <%= f.text_field :email, placeholder: "example@example.com" %></br>
+  Password:</br> <%= f.text_field :password, placeholder: "Your password" %></br>
+  </br><%= f.submit 'Submit'%></br>
+<% end %>
+
+
+
+<!--
+<form accept-charset="UTF-8" action="/users" method="post">
```

Comment:
It's better to have `users_path` instead of hardcoding the address `/users`. The first one will always direct the user to the right place even if the address changes.